incident_viewer_demo/src/LineChart/LineChart.jsx

```diff
@@ -280,10 +280,10 @@ class LineChart extends PureComponent {
    * @return {void}
    */
   onMouseMove(mouse) {
-    const { plotAreaHeight, goodIncidentSeries, badIncidentSeries, onHighlightDate, series, xScale , xKey, yScale, hasIncident } = this.props;
-    
-    const goodDescription1 = "Average Download Speed: 50 mb/s"
-    const goodDescription2 = "August 2015 - July 2016"
+    const { plotAreaHeight, incident, hasIncident, onHighlightDate, series, xScale , xKey, yScale, yKey } = this.props;
+    const goodDescription1 = incident.goodPeriodInfo
```

Comment:
Could we get away without the '1' at the end of these var names?

Reply:
Done!